src/libraries/System.Text.Json/tests/TrimmingTests/CollectionConverters/CollectionConvertersTest.IImmutableDictionary.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Text.Json;
+
+namespace SerializerTrimmingTest
+{
+    /// <summary>
+    /// Tests that (de)serializing IImmutableDictionary<TKey, TValue> is linker-safe.
```

Comment:
(nit) why are all the Immutable* types split into their own tests, but all the other collections are all slammed together into 1 test?

Reply:
I've split all the collection types into their own tests.